platform/security/security-review-process.md

```diff
@@ -0,0 +1,25 @@
+# Security review process
+
+## Context
+
+Before launching an application on VA.gov, the feature should be reviewed by a VSP security engineer.
+
+This isn't intended to be a thorough audit, but a high-level overview of your application, with a focus on any sensitive areas.
+
+**Note: this is extremely low-process today and will be more fleshed out in the future.**
+
+## Process
+
+1. File a [GitHub issue in `va.gov-team`](https://github.com/department-of-veterans-affairs/va.gov-team/issues/new):
+    * Provide the title "Security Review for [feature name]"
+    * Tag with the label `security-review`
+    * Provide the following in the ticket:
```

Comment:
Should we be clear that we intend for this documentation to live in the `va.gov-team` not the `va.gov-team` issue.

Reply:
I like that idea, reworded